src/Coin.cpp

```diff
@@ -225,11 +228,8 @@ DerivationPath TW::derivationPath(TWCoinType coin) {
     case TWCoinTypeXDai:
     case TWCoinTypeZcash:
     case TWCoinTypeZcoin:
-        return DerivationPath(purpose(coin), coin, 0, 0, 0);
-
     case TWCoinTypeAion:
-        return DerivationPath(purpose(coin), coin, 0, 0x80000000, 0x80000000);
-
+        return DerivationPath(purpose(coin), coin, 0, 0, 0);
```

Comment:
This is incorrect, please revert this change.

Reply:
revert incorrect change.

Reply:
I had changed in newest code, maybe you need to resolve conflicts first. then, my newest code you can see.

Reply:
I still see an incorrect value, unfortunately I can't approve unless you revert this change.

Reply:
`DerivationPath(purpose(coin), coin, 0, 0, 0);` is right, which is same with your repo‘s master branch: https://github.com/TrustWallet/wallet-core/blob/master/src/Coin.cpp#L262

Now, my newest commit also this one: https://github.com/TrustWallet/wallet-core/pull/103/commits/db8130f5fa2068434da20f4919abac1f10b37717